alpine-nix-rails-nginx/etc/sv/rails

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -o errexit -o pipefail -o noglob -o noclobber -o nounset
+IFS=$'\n\t'
+
+source /etc/profile.d/nix.sh
```

Comment:
Actually, I wonder if this is necessary now that the code is being executed within the context of the shell. I will test removing this.